### ISSUE TITLE
json response to "actions"

### ISSUE THREAD

**Issue by sebastianelsner** [CGRU/cgru#217]
With the newest version of afanasy (2.1) sending an action to the server results in a {"info":{"kind":"log","text":"JSON message pushed to run queue."}} message response. Previously (2.0.8) one would get a tuple like (True, {"status":"OK"}). did this tuple ever have meaning, like "yes, your action was executed ok"? If so, how would I know this now?

**Comment by timurhai** [comment]
Hi.
For now, server architecture can't return action status. And 2.0.8 could not do too. Just answer message changed.
When server see an action message it push it to queue for the main thread.

 {"status":"OK"} - was added recently too. Previously server just close socket with no answer at all. But for performance reasons, we decided that server should return something in any case, and waits client to close socket - this prevents TIME_WAIT sockets growth on server side ( as it happens on one connection side -  who calls close() first )

Later we make only 1 messages queue. And its already done by
https://github.com/eliemichel/cgru/commits/socketpool
But for now i want to test first pack of changes, that were caused by "deliver-less" server architecture.
